uui/components/filters/PresetPanel/PresetInput.module.scss

```diff
@@ -20,21 +20,34 @@
 
         :global(.uui-icon-accept) {
             &:global(.-clickable) {
-                fill: var(--white);
                 background-color: var(--uui-accent-50);
+                border-radius: var(--uui-border-radius);
+                fill: var(--white);
```

Comment:
May be better use for such places --uui-accent-contrast instead of --white?

Reply:
Better to change IconContainers to Button